posthog/api/cohort.py

```diff
@@ -81,6 +81,79 @@
 )
 from posthog.utils import format_query_params_absolute_url
 from prometheus_client import Counter
+from typing import Literal, Annotated
+from pydantic import BaseModel, Field, model_validator
+from pydantic import ValidationError as PydanticValidationError
+
+
+class EventPropFilter(BaseModel, extra="forbid"):
+    type: Literal["event"]
+    key: str
+    value: Any
```

Comment:
Shouldn't this be `Any | None` for the `not_set` case, or does that not apply to event property filters?

Reply:
event properties also have values (different for person properties)